it/mail/chrome/messenger/addressbook/abMainWindow.dtd

```diff
@@ -148,20 +148,20 @@
 <!ENTITY throbberItem.title                             "Indicatore di attività">
 
 <!-- Address Book Toolbar Tooltips -->
-<!ENTITY newcardButton.tooltip                          "Create a new address book card">
+<!ENTITY newcardButton.tooltip                          "Crea un nuovo contatto in rubrica">
 <!ENTITY newlistButton.tooltip                          "Crea una nuova lista">
-<!ENTITY editButton.tooltip                             "Edit the selected card">
-<!ENTITY newmsgButton.tooltip                           "Invia un messaggio di posta">
+<!ENTITY editButton.tooltip                             "Modifica il contatto selezionato">
+<!ENTITY newmsgButton.tooltip                           "Invia un messaggio e-mail">
 <!ENTITY newIM.tooltip                                  "Invia un messaggio istantaneo o avvia la chat">
-<!ENTITY deleteButton.tooltip                           "Delete selected card or address book">
+<!ENTITY deleteButton.tooltip                           "Elimina il contatto o la rubrica selezionati">
 
 <!-- Dir Tree header -->
 <!ENTITY dirTreeHeader.label                            "Rubriche">
 
 <!-- Search Bar -->
 <!ENTITY SearchNameOrEmail.label                        "Nome o e-mail">
 <!ENTITY SearchNameOrEmail.accesskey                    "N">
-<!ENTITY advancedButton.tooltip                         "Ricerca avanzata">
+<!ENTITY advancedButton.tooltip                         "Ricerca avanzata indirizzo">
```

Comment:
CORRECTIONS NEEDED? "Indirizzo" isn't present at all in the first version. Please check the source text.